CONTRIBUTING.md

```diff
@@ -2,8 +2,186 @@
 The Hubs documentation site has several different types of content that covers both Hubs and Spoke. Please keep the following structure in mind when creating new documentation files for the site. 
 
 * Getting Started Guides
+  - Guide    - Getting started with Github
```

Comment:
I'd prefer if we linked to a guide that's already been written rather than include one here. 

The content below seems to have come from 
[How To Contribute to Open Source: Getting Started with Git](https://www.digitalocean.com/community/tutorials/how-to-contribute-to-open-source-getting-started-with-git)
and 
[How To Create a Pull Request on GitHub](https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github) . 
I'm not comfortable taking the text (especially without attribution) but would be OK with inserting links to these instead. 

Other valuable resources might be [the Git book](https://git-scm.com/book/en/v2) and [github's documentation](https://docs.github.com/en/free-pro-team@latest/github)